Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:slim
+MAINTAINER Geppe Brillo
+RUN mkdir -p /blablabla/logs /blablabla/bots
+WORKDIR /blablabla
+COPY . ./
+RUN pip install -r dev-requirements.txt
+RUN chown -R nobody: /blablabla
+USER nobody
+RUN echo '#!/bin/bash \nWORKDIR="/blablabla"\nexport BLABLABLA_LOGGING_CONF=${WORKDIR}/logging.dev.conf\nGUNICORN_CMD_ARGS="--bind=0.0.0.0:8000 --workers=1 --access-logfile ${WORKDIR}/logs/access.log --error-logfile ${WORKDIR}/logs/error.log" BLABLABLA_LOGGING_CONF="${WORKDIR}/logging.conf" gunicorn "api:from_config(\"config.toml\")"' > ./entrypoint.sh && chmod +x ./entrypoint.sh
```

Comment:
Did you consider starting the service via CMD` (or copying an `entrypoint.sh` script to the image) instead of creating the script via I/O redirect?

Reply:
I did, I preferred to keep al docker related config into Dockerfile as it's pretty limited.